packages/contracts/utils/ens.ts

```diff
@@ -17,6 +20,7 @@ export async function setupENS(domains: string[], hre: EHRE): Promise<any> {
 
   // Deploy the ENSRegistry
   await deploy('ENSRegistry', {
+    contract: ensRegistryArtifact,
     from: deployer.address,
```

Comment:
Not the PR's focus, but shouldn't this function belong to a path like `packages/contracts/deploy` instead of utils?